CHANGELOG.md

```diff
@@ -6,6 +6,17 @@
 
 ## 2.25.0
 
+### New Features
+- **AWSIoT**
+  - AWSIoT now supports retained messages ([PR #3681](https://github.com/aws-amplify/aws-sdk-ios/pull/3681))
+
+### Breaking Changes
+  - IoT: updates nullability status for ackCallback for publishing messages
+
+* Marks enumerate blocks methods as not available and provides and alternative which does work with Swift. (See [PR #3726](https://github.com/aws-amplify/aws-sdk-ios/pull/3726))
```

Comment:
Please follow the styling in the document:

```
**AWSS3TransferUtility**
Marks enumerate blocks methods as not available and provides and alternative which does work with Swift. (See [PR #3726](https://github.com/aws-amplify/aws-sdk-ios/pull/3726))
```

Is this a breaking change now?